frontend/src/metabase/containers/UndoListing.jsx

```diff
@@ -66,7 +66,7 @@ function UndoToast({ undo, onUndo, onDismiss }) {
         >
           <CardContent>
             <CardContentSide>
-              <CardIcon name={undo.icon || "check"} color="white" />
+              {undo.icon && <CardIcon name={undo.icon} color="white" />}
```

Comment:
if we have an action with always specified undo icon as "check", does it mean we still have a case when `icon` can be nullable?

Reply:
yes, because we use `null` to remove the default icon